src/components/global/Playground/stackblitz.utils.ts

```diff
@@ -130,13 +130,14 @@ const openAngularEditor = async (code: string, options?: EditorOptions) => {
 }
 
 const openReactEditor = async (code: string, options?: EditorOptions) => {
-  let [index_tsx, app_tsx, variables_css, ts_config_json, package_json, package_lock_json] = await loadSourceFiles([
+  const [index_tsx, app_tsx, variables_css, ts_config_json, package_json, package_lock_json, index_html] = await loadSourceFiles([
```

Comment:
```suggestion
  let [index_tsx, app_tsx, variables_css, ts_config_json, package_json, package_lock_json, index_html] = await loadSourceFiles([
```

Changing this to a `const` causes an error with line 143:

```
app_tsx = app_tsx.replace('{{ MODE }}', options?.mode);
```

and a console error when trying to open a React playground: 

```
Uncaught (in promise) TypeError: "app_tsx" is read-only
```